index.html

```diff
@@ -75,133 +77,137 @@ <h1 class="hero-heading">Build better processes</h1>
     <!-- trending technologies-->
     <section class="trending-technologies-section">
         <!-- <div class="container-common"> -->
-            <div class="trending-technologies-main-container">
-                <div class="trending-technologies-text-content">
-                    <h6 class="title-pink">TRENDING TECHNOLOGIES</h6>
-                    <h3 class="title-sub">Popular topics
-                        to learn now</h3>
-                    <p>
-                        Not sure what technologies to focus on? The Technology Index ranks 850+ technologies by their
-                        relative
-                        popularity. See what’s topping the charts and get recommended courses to skill up.
-                    </p>
-                    <div class="arrow-path-container">
-                        <div class="arrow-path-text">
-                            <p>View the index</p>
-                        </div>
-                        <div class="right-arrow">
-                            <span><img src="./images/chevron-right.svg" alt=""></span>
-                        </div>
+        <div class="trending-technologies-main-container">
```

Comment:
Just "trending-technologies-container" is fine

Reply:
done